### ISSUE TITLE
Only IOS it doesn't work when app is killed

### ISSUE THREAD

**Issue by insoon0930** [rekabhq/background_locator#291]
i just found out that latest version(1.6.6) example doesn't work in IOS when app is killed... (it works in android perfectly)

https://user-images.githubusercontent.com/74125508/143516181-454c3711-44ba-47f7-99ac-b4fce68ee205.mov

(real device, iphone 11, IOS 15.1)

#54 Reading this issue, seems like you guys solved it, but is there anything else i need to do? I'm confused because it doesn't work...

**Comment by andrewzakhartchouk** [comment]
Can you share the version of background_locator in your pubspec.yaml
Also the code that starts the loaction service?

**Comment by insoon0930** [comment]
> Can you share the version of background_locator in your pubspec.yaml Also the code that starts the loaction service?

first, thanks for the reply :)
the version is 1.6.6 and i used exactly same code with the 'plugin's example' by copy and paste..
does it work on your IOS device when it is killed? 
now i'm wondering am i the only one struggling this

**Comment by NJashwanth** [comment]
Even I am facing the same issue. Did you find any solution? I am using 1.6.6 version and working with example provided by Plugin.

**Comment by insoon0930** [comment]
> Even I am facing the same issue. Did you find any solution? I am using 1.6.6 version and working with example provided by Plugin.

still the same :(

**Comment by andrewzakhartchouk** [comment]
I could get it to work on both platforms, on emulators and real devices. Have you followed all the instructions in the wiki?

**Comment by signalmeeting** [comment]
> I could get it to work on both platforms, on emulators and real devices. Have you followed all the instructions in the wiki?

yes I did, and it looks like other people having the same issue on IOS as you see on Issues ㅜㅡㅜ

**Comment by insoon0930** [comment]
> I could get it to work on both platforms, on emulators and real devices. Have you followed all the instructions in the wiki?

yes I did, and it looks like other people having the same issue on IOS as you see on Issues ㅜㅡㅜ
